src/assets/assets.service.ts

```diff
@@ -165,6 +166,28 @@ export class AssetsService {
     };
   }
 
+  listIdentifiers(options: ListAssetIdsOptions): Promise<Partial<Asset>[]> {
```

Comment:
```suggestion
  async listIdentifiers(options: ListAssetIdsOptions): Promise<Partial<Asset>[]> {
```

Reply:
This is returning a `Promise` directly without using any `await` expression; it shouldn't be marked `async`